src/test/java/stringcalculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,86 @@
+package stringcalculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+public class StringCalculatorTest {
+
+    @Test
+    @DisplayName("덧셈 테스트")
+    void testPlus() {
+        //arrange
+        String input = "1 + 4 + 7";
```

Comment:
```suggestion
        String input = "12;
```

덧셈이 어떻게 되는지 과정을 넣을 필요는 없어보여요.
테스트 코드는 단지 `StringCalculator.calculate(input);` 메소드 호출할때 `input`에 대한 `output`이 궁금할 뿐이라고 생각하기 때문이에요.

테스트 코드는 단순하게 유지해주는것이 좋아요!